src/main/java/seedu/address/model/person/UniqueCcaList.java

```diff
@@ -0,0 +1,181 @@
+package seedu.address.model.person;
```

Comment:
If you copied this package from another part of the folder, you need to credit it.

Reply:
is there a guideline for this? because it is modified quite a bit, and i did consider making UniqueList a generic for this.

Reply:
My personal guideline is if I ctrl +c-ed some part i just credit it. Doesn't hurt to do so even if its just 10% of the code.

Reply:
That's true, I would usually do that for other projects; I'll do this tmr. I was just under the impression from ur comment there was some kinda guideline we have to follow for this module.

Reply:
I've addressed this.